src/lib/web3-utils.js

```diff
@@ -0,0 +1,39 @@
+import env from '../environment'
+
+export function getUseWalletProviders() {
+  const providers = [{ id: 'injected' }, { id: 'frame' }]
+
+  if (env('FORTMATIC_API_KEY')) {
+    providers.push({
+      id: 'fortmatic',
+      useWalletConf: { apiKey: env('FORTMATIC_API_KEY') },
+    })
```

Comment:
Wondering if the idea is to add more things here, or if we should define a useWallet()-compatible object directly?

Reply:
There's more to come here, I think i'll remove this for now and add as part of a separate PR for the wallet